core/src/main/java/org/apache/iceberg/DeleteFileIndex.java

```diff
@@ -377,7 +377,7 @@ Builder planWith(ExecutorService newExecutorService) {
     }
 
     DeleteFileIndex build() {
-      // read all of the matching delete manifests in parallel and accumulate the matching files in a queue
+      // read all the matching delete manifests in parallel and accumulate the matching files in a queue
```

Comment:
Nit: Can we avoid making unnecessary changes to files?

Every additional unneeded diff makes the work of cherry-picking individual commits into private forks that much more difficult for those who maintain their own company-specific forks.